app/routes.py

```diff
@@ -9,11 +9,12 @@
 @app.route('/index', methods=['GET'])
 def index():
     #Challenges
-    challenges = []
-    for chal in getAllChallenges():
-        challenges.append(chal.toDict())
+    challengeList = []
+    challenges = Challenge.query.all()
+    for challenge in challenges:
+        challengeList.append(challenge.toDict())
```

Comment:
To my understanding, everything here looks excellent.

One (very) small thing is that this refactor has eliminated the secondsToTime utility function and instead has timeLeft as an Epoch integer.

Are you able to add something here that, when looping through these challenges, uses that utility instead.

LGTM other than that though

Reply:
Done :)

It no longer shows an epoch integer, but the amount of time remaining that's actually readable by a human.

![image](https://github.com/RedBlueCarrots/WikiPath/assets/67453536/0f37a649-b931-4e18-8a64-5091ee6f8d43)

(May have to look into the order of which challenges are displayed...)

Reply:
Order we can worry about later in MMP, LGTM! Happy to approve